docker/test/stateless/run.sh

```diff
@@ -143,18 +160,17 @@ pigz < /var/log/clickhouse-server/clickhouse-server.log > /test_output/clickhous
 #   for files >64MB, we want this files to be compressed explicitly
 for table in query_log zookeeper_log trace_log transactions_info_log
 do
-    clickhouse-client -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.tsv.gz &
```

Comment:
The problem with using `clickhouse-local` here is that now it is not capable to save data from S3 storage, like the comment above this lines states. I guess `clickhouse-local` can be taught to use S3 (maybe simply using server config will be enough, though not sure)

Reply:
https://github.com/ClickHouse/ClickHouse/pull/54198
https://github.com/ClickHouse/ClickHouse/pull/54246